features/telemetry/protocols/graphite/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/graphite/adapter/GraphiteAdapter.java

```diff
@@ -95,12 +96,12 @@ public GraphiteAdapter(final AdapterDefinition adapterConfig, final MetricRegist
             if (entry.length != 3) {
                 LOG.warn("Unparseable graphite plaintext message: {}", line);
             } else {
-                final GraphiteMetric metric = new GraphiteMetric(entry[0], entry[1], Long.valueOf(entry[2], 10));
                 try {
+                    final GraphiteMetric metric = new GraphiteMetric(entry[0], entry[1], Long.valueOf(entry[2], 10));
```

Comment:
The `NumberFormatException`  still won't get caught here right?

Reply:
d'oh, you're right